### DISCUSSION TITLE
Spacemacs cloning issue

### DISCUSSION THREAD

**Discussion by ttamttam** [community/community#126713] [New to GitHub] [answered]
`git clone https://github.com/syl20bnr/spacemacs.git local_dir` gives me works, but retrieves me an 8 weeks-old version of the default branch.

I do not understand what I am doing wrong?

**Comment by ttamttam** [comment, marked as answer]
My bad: sorry for the noise.

**Reply by Saud9911** [reply]
H

**Comment by Saud9911** [comment]
> `git clone https://github.com/syl20bnr/spacemacs.git local_dir` gives me works, but retrieves me an 8 weeks-old version of the default branch.
> 
> 
> 
> I do not understand what I am doing wrong?

H
